plugins/modules/api_modify.py

```diff
@@ -228,6 +228,11 @@
       - remove
       - remove_as_much_as_possible
     default: ignore
+  handle_stratified:
+    description:
+        - TODO
+    type: bool
+    default: false
```

Comment:
How about instead allowing to specify which keys (for stratification) to consider, so that only entries for these keys are accepted by the module and that only entries for these keys are considered?